src/core/meshcop/network_name.cpp

```diff
@@ -34,6 +34,7 @@
 
 #include "network_name.hpp"
 
+#include "common/string.hpp"
```

Comment:
Suggest removing the `#include "common/string.hpp"`? It should already be included indirectly via `instance.hpp`.

I recently worked on simplifying header includes in #10717, with the goal of removing other headers in `cpp` files when `instance.hpp` is included (since it typically includes everything else necessary). This helps speed up build times (a little bit) and might even speed it more if/when we enable pre-compiled headers.

Reply:
Ok, done now.